rust/src/labels.rs

```diff
@@ -25,11 +25,11 @@ pub struct Label {
 }
 
 impl Label {
-    pub fn from_row(row: &Vec<TypedValue>) -> Option<Label> {
+    pub fn from_row(row: &Vec<Binding>) -> Option<Label> {
         Some(Label {
-            id: row[0].clone().to_inner(),
-            name: row[1].clone().to_inner(),
-            color: row[2].clone().to_inner()
+            id: row[0].clone().val().expect("typed value").to_inner(),
```

Comment:
I think this is the source of your failing `Label` tests. It seems that `id` is no longer a part of the schema (see `initialize`) - we have only `:label/name` and `:label/color`. In `fetch_label` we can also see we are only requesting `:label/name` and `:label/color` and yet we can see in `labels.rs` we are fetching three values from the result set. This is almost certainly the cause.